damnit/api.py

```diff
@@ -19,6 +20,7 @@ class DataType(Enum):
     Dataset = "dataset"
     Image = "image"
     Timestamp = "timestamp"
+    PlotlyFigure = "PlotlyFigure"
```

Comment:
Nitpick: `PlotlyJson = "plotly_json` perhaps?

Reply:
I prefer `PlotlyFigure` as this is the object that is to be saved, and returned by the API.